index.rst

```diff
@@ -697,14 +669,11 @@ Non-Configurable CSCs
 
 Some CSCs will not be configurable at all.
 Examples are sparse in our current architecture but, the from Salobj point of view, a CSC can be developed on top of a ``BaseCSC`` which makes it a non-configurable component.
+Non-configurable CSCs will have no data in the configuration column of the `Master CSC Table`_.
 
-A non-configurable CSC will ignore the ``configurationToApply`` attribute of the ``start`` command, as it does not contain any true meaning to it.
+A non-configurable CSC will ignore the ``configurationOverride`` attribute of the ``start`` command, as it does not contain any true meaning to it.
```

Comment:
Please consider "as it has no meaning".

Reply:
Implemented, but is this the proper behaviour? Shouldn't it reject the command saying it can't be configured?
I suppose it could also say it can't find the file, but that would probably send the user down a rabbit hole.